src/ace/managers/memory.c

```diff
@@ -144,7 +144,7 @@ void _memDestroy(void) {
 
 void *_memAllocDbg(ULONG ulSize, ULONG ulFlags, UWORD uwLine, char *szFile) {
 	void *pAddr;
-	pAddr = _memAllocRls(ulSize, ulFlags);
+	pAddr = _memAllocRls(ulSize + 2*sizeof(ULONG), ulFlags);
```

Comment:
that's not something we agreed upon in our last codestyle issue - we'll have this discussion somewhere else at the later time. I'll add missing spaces instead since IMO brackets would decrease code readability